services/claudeAi.js

```diff
@@ -0,0 +1,25 @@
+const Anthropic = require("@anthropic-ai/sdk");
+require('dotenv').config();
+
+const anthropic = new Anthropic({
+    apiKey: process.env['ANTHROPIC_API_KEY'], // This is the default and can be omitted
+});
+
+
+async function generateClaudeContent(content, model_type = "claude-3-5-sonnet-20240620") {
+    try {
+        const result = await anthropic.messages.create({
+            max_tokens: 4096,
```

Comment:
can you remove this and so it will use the default?

Reply:
This is the maximum